syntax/ocaml.vim

```diff
@@ -109,6 +109,10 @@ endif
 " "if"
 syn region   ocamlNone matchgroup=ocamlKeyword start="\<if\>" matchgroup=ocamlKeyword end="\<then\>" contains=ALLBUT,@ocamlContained,ocamlThenErr
 
+"" PPX nodes
+
+syn region ocamlPpx matchgroup=ocamlPpxEncl start="\[@@\w\+\(.\w\+\)*" contains=TOP end="\]"
```

Comment:
`start` should probably use something like `@\{1,3\}` to consolidate this down to one line and support `[@@@attr]`.

Reply:
More concretely:
```
start="\[@\{1,3\}\w\+\(.\w\+\)*"
```

Reply:
Incorporating @copy's `.` fix:
```
start="\[@\{1,3\}\w\+\(\.\w\+\)*"
```
which avoids highlighting syntax like `[@attr*foo]`.